bindings/GtkSheet/GtkSheet.overrides

```diff
@@ -0,0 +1,15 @@
+namespace GtkSheet
+
+# Conflicts with a method of same name on GtkSheet/Sheet
+delete-node GtkSheet/SheetColumn/visible
+
+# Generated from gtksheet 4.3.2 with xsltproc Nullable.xslt GtkSheet-4.0.gir
```

Comment:
Unless you specifically want to support old versions in which there annotations were not present, you can remove these `set-attr` to make the overrides file a bit simpler. (They are no-ops if the nullable annotations are already there, so from version `4.3.2` onwards at least, and possibly earlier.)